src/playground-code-editor.ts

```diff
@@ -495,6 +505,20 @@ export class PlaygroundCodeEditor extends LitElement {
       }
     });
 
+    cm.on('renderLine', (_editorInstance: Editor, line, elt) => {
+      if (!this.lineWrapping) {
```

Comment:
Sorry, for some reason I didn't notice this part of the code when I first looked at this PR.

Why do we need special handling here? Why doesn't the built-in `lineWrapping` option work as-is?

Reply:
Do you think we could add and remove the `renderLine` listener all together whenever `lineWrapping` changes inside `update`, instead of always listening and then aborting? Should be more efficient in the case where it is disabled.